cli/test/all_command_permutation_create_csv.dart

```diff
@@ -0,0 +1,238 @@
+import 'dart:io';
+
+void main() {
+  // Create CSV file
+  final file = File('command_permutations.csv');
+  final sink = file.openWrite();
+
+  // Write CSV header
+  sink.writeln('command,flags,arguments,description,full_command');
+
+  final staxCommands = [
```

Comment:
Let's use commands defined here: https://github.com/TarasMazepa/stax/blob/main/cli/lib/command/internal_commands.dart#L22 , but filter only to allow public.